### DISCUSSION TITLE
How to use multi-process inference on PaddleOCR?

### DISCUSSION THREAD

**Discussion by lijzeng** [PaddlePaddle/PaddleOCR#6018] [Q&A] [answered]
How to use multi-process inference on PaddleOCR?

**Comment by Evezerest** [comment, marked as answer]
PaddleOCR added [Control parameters of multi-process inference](https://github.com/PaddlePaddle/PaddleOCR/blob/8b656a3e13631dfb1ac21d2095d4d4a4993ef710/tools/infer/utility.py?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L111). `use_mp` indicates whether there are multiple processes, and `total_process_num` represents the number of processes. For detailed usage, please refer to [document](https://github.com/PaddlePaddle/PaddleOCR/blob/release%2F2.3/doc/doc_ch/inference.md#1-%E8%B6%85%E8%BD%BB%E9%87%8F%E4%B8%AD%E6%96%87ocr%E6%A8%A1%E5%9E%8B%E6%8E%A8%E7%90%86).
